codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ActionServiceSourceGenerator.scala

```diff
@@ -294,7 +294,7 @@ object ActionServiceSourceGenerator {
         |}
         |
         |class ${service.providerName} private(actionFactory: ActionCreationContext => ${service.className},
-        |                                      options: ActionOptions)
+        |                                      val options: ActionOptions)
         |  extends ActionProvider[${service.className}] {
         |
```

Comment:
add `def withOptions` while you are at it

Reply:
it's there already, line 310

Reply:
ah, thanks